libfreerdp/core/input.c

```diff
@@ -222,6 +223,46 @@ static BOOL input_send_mouse_event(rdpInput* input, UINT16 flags, UINT16 x, UINT
 	return rdp_send_client_input_pdu(rdp, s);
 }
 
+static BOOL input_send_relmouse_event(rdpInput* input, UINT16 flags, INT16 x, INT16 y)
+{
+	wStream* s;
+	rdpRdp* rdp;
+
+	if (!input || !input->context || !input->context->settings)
+		return FALSE;
+
+	rdp = input->context->rdp;
+
+	if (!input_ensure_client_running(input))
+		return FALSE;
+
+	if (!freerdp_settings_get_bool(input->context->settings, FreeRDP_HasRelativeMouseEvent))
+	{
+		WLog_ERR(TAG, "Sending relative mouse event, but no support for that");
+		return FALSE;
+	}
+
+	if (!freerdp_settings_get_bool(input->context->settings, FreeRDP_HasHorizontalWheel))
+	{
+		if (flags & PTR_FLAGS_HWHEEL)
+		{
+			WLog_WARN(TAG,
+			          "skip mouse event %" PRIu16 "x%" PRIu16 " flags=0x%04" PRIX16
```

Comment:
`skip mouse event` -> `skipping relative mouse event`

Reply:
copy & paste error, wheel events are not allowed